pkg/aws/core.go

```diff
@@ -141,6 +145,7 @@ func (d *Driver) CreateMachine(ctx context.Context, req *driver.CreateMachineReq
 		},
 		NetworkInterfaces: networkInterfaceSpecs,
 		TagSpecifications: []*ec2.TagSpecification{tagInstance, tagVolume},
+		Placement:         getPlacementObj(req),
 	}
 
```

Comment:
```suggestion
		NetworkInterfaces: networkInterfaceSpecs,
		TagSpecifications: []*ec2.TagSpecification{tagInstance, tagVolume},
	}
	if placement := getPlacementObj(req); placement != nil {
	  inputConfig.Placement =  placement
	}
```
If you take my suggestion, the code would look something like this. This way current code/requests are not affected.

Reply:
made these changes, thanks for the suggestion